pkg/cmd/server/kubernetes/master.go

```diff
@@ -107,6 +115,24 @@ func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string
 	recycler.Run()
 }
 
+// attemptToLoadRecycler tries decoding a pod from a filepath for use as a recycler for a volume.
+// If a path is not set as a CLI flag, no load will be attempted and no error returned.
+// If a path is set and the pod was successfully loaded, the recycler pod will be set on the config and no error returned.
+// Any failed attempt to load the recycler pod will return an error.
+// TODO: make this func re-usable upstream and use downstream.  No need to duplicate this function.
+func attemptToLoadRecycler(path string, config *volume.VolumeConfig) error {
+	glog.V(5).Infof("Attempting to load recycler pod file from %s", path)
```

Comment:
this won't make sense if they haven't provided a value... I'd rather see us do the length check in RunPersistentVolumeClaimRecycler and only call this function if a path was provided